api/src/test/java/uk/gov/dhsc/htbhf/claimant/ClaimantLifecycleIntegrationTests.java

```diff
@@ -267,6 +307,12 @@ private void ageByOneCycle() {
         invokeAllSchedulers();
     }
 
+    private void ageByeNumberOfCycles(int numberOfCycles) {
+        for (int i = 0;i < numberOfCycles;i++) {
+            ageByOneCycle();
+        }
+    }
+
     private void ageByFourCycles(List<LocalDate> initialChildrenDobs) throws JsonProcessingException {
```

Comment:
Maybe remove this method now?

Reply:
It's a different method, it takes in the children's dob and modifies them as well. I've updated it to take a number of cycles as a parameter.